### ISSUE TITLE
Develop tutorial for oncology extension.

### ISSUE THREAD

**Issue by mgurley** [OHDSI/OncologyWG#121]
Dependent on #22

**Comment by sratwani** [comment]
@rimusia Can I assign this to you based on our conversation around documentation.

**Comment by sratwani** [comment]
Per Leadership call on 12/16, the group decided that this task is out of scope for the milestone 'Rerun symposium plots'. Moving it to the backlog for now.
